tests/test_vector_search.py

```
@@ -0,0 +1,385 @@
+# tests/test_vector_search.py
+import pytest
+import sys
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)